source/content/drupal-9.md

```diff
@@ -0,0 +1,91 @@
+
+1. Spin up a new "Drupal 9 Early Access" site with `ic-demo-2020-` as a site name prefix (i.e., `ic-demo-2020-beta3-d9-test`). This will give you [Drupal 9 starter codebase](https://github.com/stevector/drupal-9-project) and will automatically trigger a Composer install and `git commit` on site creation.
+
+## Create a Drupal 9 Site with Continuous Integration via Build Tools
+
+Want to regularly develop with continuous integration (CI) and automated tests? Leverage Pantheon’s [Build Tools](/guides/build-tools) to get up and running quickly with a CI workflow to build, test, and deploy Drupal 9. These steps use GitHub and CircleCI:
+
+1. Install the [Terminus Build Tools plugin](https://github.com/pantheon-systems/terminus-build-tools-plugin).
+   - This will require generating a Pantheon machine token, a CircleCI API token, and a GitHub API credential.
+1. Enable support for custom CircleCI Orbs in your Organizational settings. In CircleCI, click **Settings**, then **Security**, and click **Yes** to allow uncertified orbs:
+
+   ![Allow Uncertified Orbs in CircleCI](../images/circleci/circleci-allow-uncertified-orbs.png)
+
+1. Use Terminus to create a new Drupal 9 site, complete with a customizable CI workflow. Remember to replace `<NEW-SITE-NAME>` in this example with a name for your new site, and `optional-pantheon-organization-name` with your choice (if any):
+
+  ```bash{promptUser: user}
+  terminus build:project:create "pantheon-systems/drupal-9-with-circleci-orb:dev-master" <NEW-SITE-MACHINE-NAME> --stability=dev --team='optional-pantheon-organization-name'
+  ```
+
+## Test an Existing Drupal Site for Drupal 9 Upgrade Compatibility
+
+1. Review the [How to Prepare Your Drupal 7 or Drupal 8 Site for Drupal 9](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9) guide on Drupal.org.
+1. Use the [Upgrade Status Drupal 8/9 module](https://www.drupal.org/project/upgrade_status) to generate a full report of your site’s compatibility.
+1. Check out [Acquia’s Drupal 9 Deprecation Status Upgrade Tracker](https://dev.acquia.com/drupal9/deprecation_status) for information about Drupal 9 support for contributed Drupal modules and themes.
+
+### Update Deprecated Code for Drupal 9 Compatibility
+
+Drupal 9 has deprecated a number of different functions and APIs in favor of better options going forward.
+
+For example, `node_load()` was replaced in Drupal 9 with `Node::load` resulting in this change needing to be made:
+
+Drupal 8:
+
+```none
+$node = node_load(1);
+```
+
+Drupal 9:
+
+```none
+use \Drupal\node\Entity\Node;
+$node = Node::load(1);
+```
+
+Since most of these changes are relatively minor, there are a number of [deprecation checking and correction tools](https://www.drupal.org/docs/9/how-to-prepare-your-drupal-7-or-8-site-for-drupal-9/deprecation-checking-and-correction-tools) available.
+
+## FAQ
```

Comment:
Does Pantheon have plans to upgrade MariaDB?

Yes, Pantheon has plans to upgrade MariaDB platform-wide before September 30, 2020.